pkg/util/spd.go

```diff
@@ -312,6 +312,27 @@ func InsertSPDBusinessIndicatorStatus(status *apiworkload.ServiceProfileDescript
 	status.BusinessStatus = append(status.BusinessStatus, *serviceBusinessIndicatorStatus)
 }
 
+func InsertSPDAggMetrics(status *apiworkload.ServiceProfileDescriptorStatus,
+	serviceAggPodMetrics *apiworkload.AggPodMetrics,
+) {
+	if status == nil || serviceAggPodMetrics == nil {
+		return
+	}
+
+	if status.AggMetrics == nil {
+		status.AggMetrics = []apiworkload.AggPodMetrics{}
+	}
+
+	for i := range status.AggMetrics {
+		if status.AggMetrics[i].Scope == serviceAggPodMetrics.Scope {
+			status.AggMetrics[i].Aggregator = serviceAggPodMetrics.Aggregator
```

Comment:
The scope and aggregator can identify unique aggMetrics, so it only need override the Items when the scope and aggregator is equal.